src/pages/playlistpage/PlaylistPage.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useEffect, useState } from "react";
+import { VideoCard } from "../../components";
+import { usePlaylist } from "../../context";
+import AddPlaylistModal from "./AddPlaylistModal";
+
+const styleObj = {
+    backgroundColor: "var(--PRIMARY-200)",
+};
+
+export default function PlaylistPage() {
```

Comment:
why are you not using aero function?

Reply:
I personally like to prefer normal function for Components